projects/arlas-toolkit/src/lib/components/timeline/timeline/timeline.component.ts

```diff
@@ -140,18 +146,23 @@ export class TimelineComponent implements OnInit {
       this.resetHistogramsInputs(this.timelineComponent.input);
       this.timelineContributor = <HistogramContributor>this.arlasStartupService.contributorRegistry
         .get(this.timelineComponent.contributorId);
-
       const mainCollection = this.timelineContributor.collection;
+      const mainCollectionDisplayName = !!this.arlasStartupService.collectionsMap.get(mainCollection).display_names?.collection ?
```

Comment:
`mainCollection` and `mainCollectionDisplayName` can be define out of the `if/else` statement
to avoid code duplication

Reply:
Fixed